assets/app.js

```diff
@@ -922,6 +929,23 @@ new Vue({
                 api.call()
             }
         },
+        getDescriptorView: function (){
+
+            languages = this.accessRight;
+            AppLanguages.forEach(function (element) {
+                if (languages.includes(element['id'])) {
+                    element['status'] = true
```

Comment:
instead of this whole if statement, do a `element['status'] === languages.includes(element['id')`?

Reply:
@cyberhck 
you mean only this?
```
AppLanguages.forEach(function (element) {
     element['status'] === languages.includes(element['id')
});
``` 
This is not working.

Reply:
`element['status'] = languages.includes(element['id])`

Reply:
my bad, I forgot a `]`, can you try again?

Reply:
where's the `accessRight` defined? does `languages.includes` return `boolean`?

Reply:
I try this it is work!
 ```
AppLanguages.forEach(function (element) {
       element['status'] = languages.includes(element['id'])
 });
```

- `where's the accessRight defined? `  it's defined in product_descriptor.html.twig in `shop`
- ` does languages.includes return boolean? ` yes

Reply:
By the way `.includes()` is not supported by IE. Will this app.js be run on browser or node environment? If on browser, I recommend to use `.indexOf()` instead.

Reply:
AFAIK we'll not be using IE, only chrome/firefox (backend), But I'm not sure 🙂

Reply:
We don't care for IE, since we know everyone using it. Nevertheless Pai is right: use `.indexOf()`